.gitignore

```diff
@@ -16,4 +16,3 @@ npm-debug.log*
 *.log
 Thumbs.db
 .DS_Store
-.eslintcache
```

Comment:
In the latest patch version of CRA, this file has moved back to its proper location in the `node_modules/.cache` folder.